debezium-core/src/main/java/io/debezium/connector/common/TaskOffsetContext.java

```diff
@@ -26,6 +26,15 @@ public TaskOffsetContext(Map<P, O> offsets) {
         return offsets;
     }
 
+    public boolean isNull() {
+        for (O offset : offsets.values()) {
```

Comment:
This logic should be moved to the schema. If a context passed w/o a single non-empty offset, the method should log (an INFO?) and return w/o trying to read the topic.